scrapers/az/bills.py

```diff
@@ -288,26 +288,31 @@ def scrape_votes(self, bill, page):
                 action_date = datetime.datetime.strptime(
                     cleaned_date, "%Y-%m-%dT%H:%M:%S"
                 )
-                vote = VoteEvent(
-                    chamber={"S": "upper", "H": "lower"}[header["LegislativeBody"]],
-                    motion_text=action["Action"],
-                    classification="passage",
-                    result=(
+                # handle empty vote objects
+                if not action.get("Ayes", None) or not action.get("Nays", None):
```

Comment:
I think this might be the wrong logic here -- If it's missing it's not necessarily a fail. The one that's choking the scraper right now is actually a pass.

```
{'BillStatusActionID': 0, 'SessionId': 0, 'BillStatusId': 0, 'CommitteeId': 1907, 'ReferralNumber': 1, 'AssignedDate': '2022-06-23T21:00:01', 'ActionDatetime': None, 'ActionTypeId': None, 'Action': 'Passed', 'Ayes': None, 'Nays': None, 'Excused': None, 'NotVoting': None, 'Present': None, 'Absent': None, 'ReportDate': '2022-06-23T21:00:01', 'DischargeDate': None, 'MiscNotes': None, 'Vacant': 0, 'COWReferralNumber': 0, 'AdoptRevisedDate': None, 'UnanimouslyAdopted': False, 'Emergency': '', 'OverrideVeto': False, 'AdoptedDate': None, 'ChangeDate': None, 'ReportTime': None, 'MinorityReport': False, 'RevisedReport': False, 'MemberChangeDate': None, 'Amended': False, 'Recommendation': None, 'RequestForEnactment': False, 'Comments': '', 'CorrectedReport': 0, 'CorrectedDate': None, 'HouseCaucus': None, 'SenateCaucus': None, 'PostingSheet': 0, 'RFEIR': 0, 'RFEIRActionId': None, 'RFEIRSections': '', 'TwoThirdVote': 0, 'ThreeFourthVote': 0, 'ConferenceCommitteeType': None, 'ConferenceCommitteeNewType': None, 'ConferenceCommitteeNewTypeDate': None, 'RFEIRAction': '', 'CommitteeName': 'Motion to Reconsider Final', 'CommitteeShortName': 'RECON_FNL', 'Votes': []}
```

Reply:
How do we safely skip a single vote? This function `yield`s data back to openstates-core code that expects every object to have data...